src/main/scala/beam/router/r5/RoutingWorker.scala

```diff
@@ -142,6 +193,16 @@ class R5RoutingWorker(workerParams: WorkerParameters) extends Actor with ActorLo
       askForMoreWork()
 
     case UpdateTravelTimeLocal(newTravelTime) =>
+      log.info("===================================================================")
+      log.info(s"TOTAL NON TRANSIT ROUTING REQUESTS: ${routeRequestCounter.get()}, TOTAL EXECUTION TIME ${routeRequestExecutionTime.get()}, TOTAL REQUESTS ${msgs.get()}")
```

Comment:
`msgs` will be set to zero, so seems like not very useful? (https://github.com/LBNL-UCB-STI/beam/pull/2896/files#diff-5467656850d0ed2888f4e04a54e89be6R130)

Reply:
fixed